src/test/java/StringManipulationTest.java

```diff
@@ -55,95 +66,140 @@ public void testRemoveNthCharacter2() {
 
     @Test
     public void testRemoveNthCharacter3() {
```

Comment:
The following test cases (testRemoveNthCharacter 3-7) are all variations of the first two test cases. Please revise these test cases so that they're testing for different kinds of input (e.g., `n <= 0`, `string.isEmpty()`, etc.)